src/agent/multitype_queue/src/sqlitestorage.cpp

```diff
@@ -291,7 +328,7 @@ int SQLiteStorage::GetElementCount(const std::string& tableName, const std::stri
     }
     else
     {
-        constexpr std::string_view COUNT_QUERY {"SELECT COUNT(*) FROM {} WHERE module LIKE \"{}\""};
+        constexpr std::string_view COUNT_QUERY {"SELECT COUNT(*) FROM {} WHERE module_name LIKE \"{}\""};
```

Comment:
Same here.

Reply:
In this first stage, because of the way events are being sent, it is not necessary, but it should be done in the future.